<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusCylinder</class>
 <widget class="QDialog" name="MotusCylinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="handButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>41</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>252</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>254</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="subButton">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>252</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>电动缸选择</string>
   </property>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1号电动缸</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>71</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5号电动缸</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sureButton">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>38</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>71</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6号电动缸</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>21</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3号电动缸</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>21</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2号电动缸</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4号电动缸</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
